docs/assets/yamlexamples/simple-bridge.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: flow.triggermesh.io/v1alpha1
+kind: Bridge
```

Comment:
we should remove this and replace with a concatenated yaml...

Reply:
what does that mean? im sorry. @sebgoa